CONTRIBUTING.md

```diff
@@ -1,50 +1,55 @@
+your commits need to include the following line using the name and e-mail
+address you used to sign the OCA:
 
 ```text
 Signed-off-by: Your Name <you@example.org>
 ```
 
-This can be automatically added to pull requests by committing with:
+This can be automatically added to pull requests by committing with `--sign-off`
+or `-s`, e.g.
 
 ```text
 git commit --signoff
 ```
 
-Only pull requests from committers who can be verified as having signed the OCA can be accepted.
-
-### Copyright notices
-
-Please include a copyright notice consistent with other files when you introduce a new file, and edit the copyright notice of any file you change to update the list of years, separated by commas and with a comma after the last year, e.g., Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+Only pull requests from committers that can be verified as having signed the OCA
+can be accepted.
 
-It is not necessary to add your name to copyright notices because you must sign the Oracle Contributor Agreement before we can accept your contribution.  Arguments against listing all contributors in copyright notices are at the bottom of [this Linux Foundation blog post](https://www.linuxfoundation.org/en/blog/copyright-notices-in-open-source-software-projects).  
+## Pull request process
 
-### Pull request process
+1. Ensure there is an issue created to track and discuss the fix or enhancement
+   you intend to submit.
+1. Fork this repository.
+1. Create a branch in your fork to implement the changes. We recommend using
+   the issue number as part of your branch name, e.g. `1234-fixes`.
+1. Ensure that any documentation is updated with the changes that are required
+   by your change.
+1. Ensure that any samples are updated if the base image has been changed.
+1. Submit the pull request. *Do not leave the pull request blank*. Explain exactly
+   what your changes are meant to do and provide simple steps on how to validate.
+   your changes. Ensure that you reference the issue you created as well.
+1. We will assign the pull request to 2-3 people for review before it is merged.
 
-1. Fork this repository
-2. Create a branch in your fork to implement your changes. If applicable, we recommend using
-the issue number as part of your branch name, e.g. `1234-fixes`
-3. Ensure that any documentation is updated with the changes that are required
-by your fix.
-4. Ensure that your changes have not broken any existing proofs (`./Scripts/run-everything.sh yes` should complete without errors).
-5. Update the [list of contributors](README.md#contributors) to include your name if it does not already.
-6. Submit a pull request. *Do not leave the pull request blank*. Explain exactly
-what your changes are meant to do and provide simple steps on how to validate
-your changes. Ensure that you reference the issue you created as well.
-7. We will review the pull request before it is merged.
+## Code of conduct
 
-## Code of Conduct
```

Comment:
This capitalisation is inconsistent with the others you've changed, but maybe it's intentional?